service/src/ocall_bridge/sidechain_ocall.rs

```diff
@@ -86,7 +86,7 @@ where
 		if !signed_blocks.is_empty() {
 			info!(
 				"Enclave produced sidechain blocks: {:?}",
-				signed_blocks.iter().map(|b| b.block.block_number).collect::<Vec<u64>>()
+				signed_blocks.iter().map(|b| b.block.header.block_number).collect::<Vec<u64>>()
```

Comment:
Any reason why you're using `header` instead of `header()` here? (everywhere else you're using the function)